### ISSUE TITLE
TE duplicate TweakerValue for Transposer Fill/Extract

### ISSUE THREAD

**Issue by Zexks** [stanhebben/MineTweaker#10]
MineTweaker / stanhebben / minetweaker / mods / te / values / TransposerValue.java

```
public TweakerValue index(String index) {
        if (index.equals("addFillRecipe")) {
            return TransposerAddFillRecipeFunction.INSTANCE;
        } else if (index.equals("addFillRecipe")) {
            return TransposerAddExtractRecipeFunction.INSTANCE;
        } else {
            return super.index(index);
        }
```

Is it just me or should that first ELSE IF be addExtractRecipe? Looks like you can't add an extraction recipe right now.

**Comment by copygirl** [comment]
Yes, that's my fault ^^

Though, last time I tried, adding Thermal Expansion recipes doesn't work. TE needs them before postInit and MineTweaker does its thing during postInit. I don't know how we can fix this.

**Comment by Zexks** [comment]
Yeah, I'm missing some important TE recipe's in my custom pack and found this. Started googling and came across your post adding support. Noticed there was no documentation so started browsing here to decipher the commands. I tried last night and kept getting either a Parser or Tweaker exception. Not sure which, it was returning: "null:-1;exception" from right under the mod load section in the postInit (which kind of sounds like what you're talking about). So I never really got a chance to see if it just didn't read or not. Just read your post about 'fixing TE support'

Since it's not working anyways it doesn't matter much but in case I was entering them incorrectly thought I would post what I thought was the call:

modsupport.thermalexpansion.crucible.addRecipe(1000, tile.hellrock,tile.lava)
modsupport.te.crucible.addRecipe(1000, tile.hellrock,tile.lava)

And then both of those with quotes around the 1000.
